src/components/Chat/InputMessageV2.tsx

```diff
@@ -142,11 +344,11 @@ const InputMessageV2 = ({
   const sendButtonStyle = React.useCallback(() => {
     const isDisabled = isDisableButton();
     if (isDisabled) return colors.gray1;
-    if (type === 'SIGNED') return colors.darkBlue;
+    if (type === SIGNED) return colors.darkBlue;
     return colors.bondi_blue;
   }, [isDisableButton()]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendButtonStyle` function has been updated to use constants for the chat types instead of hardcoded strings. This is a good change as it reduces the risk of typos and makes the code easier to maintain.



<!-- This is an auto-generated comment by OSS CodeRabbit -->